nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryDatabaseTable.java

```diff
@@ -120,6 +120,15 @@
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor MAX_FRAGMENTS = new PropertyDescriptor.Builder()
+            .name("Maximum Number of Fragments")
```

Comment:
This is not required, but many folks have adopted a convention for new properties where name() contains a more machine-friendly name (like "qdt-max-frags" for example) and displayName() contains the user-friendly name of "Maximum Number of Fragments".